acceptance_tests/features/view_full_thread.feature

```diff
@@ -0,0 +1,33 @@
+Feature: View conversation thread
+  As an internal user
+  I need to see all of the previous messages in a conversation
+  So that I have the full context for my reply
+
+  Background: Internal user is already signed in
```

Comment:
Not sure this step is necessary anymore as the login should be disable for the acceptance tests.